apps/pigment-next-app/next.config.js

```diff
@@ -81,32 +81,20 @@ const theme = extendTheme({
       },
     },
   },
+  getSelector: function getSelector(colorScheme, css) {
+    if (colorScheme && colorScheme === 'dark') {
+      return {
+        '@media (prefers-color-scheme: dark)': {
+          ':root': css,
+        },
+      };
+    }
+    return ':root';
+  },
```

Comment:
The default `extendTheme` use `data-*` attribute for creating selectors. This custom getSelector proves that it can be customized.